app/components/menu_item/index.tsx

```diff
@@ -76,11 +87,16 @@ const MenuItem = (props: MenuItemProps) => {
         i18nId,
         iconContainerStyle,
         iconName,
+        innerContainerStyle,
```

Comment:
I see no reason why not to call this `containerStyle`

Reply:
done